packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -509,6 +504,73 @@ class PlatformZoomRange {
   final double? max;
 }
 
+class PlatformBitmap {
+  PlatformBitmap({required this.bitmap});
+
+  final Object bitmap;
```

Comment:
This needs a comment explaining why it's untyped (with a reference to https://github.com/flutter/flutter/issues/117819) and an explanation of what the valid types to set it to are.